src/core_modules/capture-core/components/WidgetEnrollment/Status/Status.component.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import React, { type ComponentType } from 'react';
+import i18n from '@dhis2/d2-i18n';
+import { withStyles } from '@material-ui/core';
+import { spacersNum, Tag } from '@dhis2/ui';
+import type { Props } from './status.types';
+
+const styles = {
+    status: {
+        margin: spacersNum.dp4,
+    },
+};
+
+export const translateStatus = (value: string) => {
+    switch (value) {
+    case 'ACTIVE': return i18n.t('Active');
```

Comment:
I think it may be better to use enum for statuses, will reduce the mis-typo. What do you think?